viewer/packages/rendering/visual-tests/RenderTarget.VisualTest.ts

```diff
@@ -11,7 +11,7 @@ import {
 import { DefaultRenderPipelineProvider } from '../src/render-pipeline-providers/DefaultRenderPipelineProvider';
 import { defaultRenderOptions } from '../src/rendering/types';
 
-export default class CadClippingVisualTest extends StreamingVisualTestFixture {
+export default class RenderTargetVisualTest extends StreamingVisualTestFixture {
```

Comment:
I assume this is just a typo from Christopher's PR? I guess it's fine to fix it here.

Reply:
Yeah, it's a type. I decided to fix it together with visual test fix.